vars/cloudFoundryDeploy.groovy

```diff
@@ -166,3 +168,10 @@ def deployMta (config) {
         sh "cf logout"
     }
 }
+
+def handleLegacyCfManifest(config) {
+    def manifest = readYaml file: config.cloudFoundry.manifest
+    manifest = CfManifestUtils.transform(manifest)
+    sh "rm ${config.cloudFoundry.manifest}"
```

Comment:
Maybe we could skip writing the file, in case nothing has changed

Reply:
As discussed, we prefer to keep it simple, and hence don't check for a change